web-client/src/components/modals/Comments.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from "react";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPaperPlane } from "@fortawesome/free-solid-svg-icons";
+import { Button, Form, Modal, Row } from "react-bootstrap/";
+import { ICommentResponse } from "../../models/comment";
+import { groupService } from "../../services/groupService";
+import { homeworkService } from "../../services/homeworkService";
+import { commentScope } from "../../shared/enums";
+
+interface CommentsProps {
+  showComments: boolean;
+  setShowComments: React.Dispatch<React.SetStateAction<boolean>>;
+  scope: commentScope;
+  scopeId: string;
+}
+
+const Comments: React.FC<CommentsProps> = ({
+  showComments,
+  setShowComments,
+  scope,
+  scopeId,
+}) => {
+  const [comments, setComments] = useState<ICommentResponse[]>([]);
+
+  useEffect(() => {
+    const fetchComments = () => {
+      switch (scope) {
+        case commentScope.GROUP: {
+          const fetchedComments = groupService.getComments(scopeId);
+          setComments(fetchedComments);
+          break;
+        }
+        case commentScope.HOMEWORK: {
+          const fetchedComments = homeworkService.getComments(scopeId);
+          setComments(fetchedComments);
+          break;
+        }
+        default:
+          break;
+      }
+    };
+
+    fetchComments();
+  }, []);
+
+  const [newComment, setNewComment] = useState<string>("");
+  const onFormControlChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setNewComment(event.target.value);
+  };
+
+  const handleClose = () => {
+    setShowComments(false);
+  };
+
+  return (
+    <div>
+      <Modal animation={false} show={showComments} onHide={handleClose}>
+        <Modal.Header closeButton>
+          <Modal.Title>Közlemények</Modal.Title>
+        </Modal.Header>
+        <Modal.Body style={{ maxHeight: "50vh", overflowY: "auto" }}>
+          {comments.map((comment: ICommentResponse) => (
```

Comment:
Lehet nem ártana egy rendezés, de mondjuk ezt a backend is leküldheti már rendezve :D